### PR TITLE
Lcam 384

### PR DESCRIPTION
**Pull request by muthusubra**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/LASB-XXX)

Describe what you did and why.

## Checklist

Before you ask people to review this PR:

- [ ] Tests should be passing: `./gradlew test`
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Comment by stone-dj** [comment]
I'm not very familiar with Liquibase so maybe this is not needed!? I can't see any indexes defined on any of the fields on any of the new tables, do we need to define indexes in these scripts or are they defined elsewhere?

File: crown-court-contribution/src/main/resources/db.changelog/changeset/02-correspondence-templates-table-create.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+--liquibase formatted sql
+--changeset muthus:02-correspondence-templates-table-create
+CREATE TABLE IF NOT EXISTS crown_court_contribution.CORRESPONDENCE_TEMPLATES
```
Comment (stone-dj) [inline crown-court-contribution/src/main/resources/db.changelog/changeset/02-correspondence-templates-table-create.sql:3]: I'm not very familiar with Liquibase so maybe this is not needed!? I can't see any indexes defined on any of the fields on any of the new tables, do we need to define indexes in these scripts or are they defined elsewhere?
(same feedback applies equally to other new tables defined in this PR also)

**Review by stone-dj — CHANGES_REQUESTED** [review]
Have left 1 comment to address or otherwise respond on.

File: crown-court-contribution/src/main/resources/db.changelog/changeset/02-correspondence-templates-table-create.sql (hunk 1 above)
Comment (muthusubra) [reply crown-court-contribution/src/main/resources/db.changelog/changeset/02-correspondence-templates-table-create.sql:3]: These are just very small static metadata tables and we don't need Indexes

**Review by mtac50 — DISMISSED** [review]

**Review by stone-dj — APPROVED** [review]
Changes look good

**Review by mtac50 — APPROVED** [review]

**Pull request merged**